src/pkg/packager/lint/lint.go

```diff
@@ -50,6 +53,61 @@ func ValidateZarfSchema(path string) (*Validator, error) {
 	return &validator, nil
 }
 
+func isPinnedImage(image string) bool {
+	if strings.HasSuffix(image, ":latest") {
+		return false
+	}
```

Comment:
I think this is redundant - we should look for digests only (since tags are never _really_ pinned)